paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PlaygroundSettingDefinition.kt

```diff
@@ -54,6 +72,27 @@ internal interface PlaygroundSettingDefinition<T> {
         }
     }
 
+    @OptIn(ExperimentalCustomerSheetApi::class)
+    data class CustomerSheetConfigurationData(
+        private val configurationBuilder: CustomerSheet.Configuration.Builder,
+        var billingDetailsCollectionConfiguration: PaymentSheet.BillingDetailsCollectionConfiguration =
+            PaymentSheet.BillingDetailsCollectionConfiguration()
+    ) {
+        // Billing details is a nested configuration, but we have individual settings for it in the
+        // UI, this helper keeps all of the configurations, rather than just the most recent.
+        fun updateBillingDetails(
```

Comment:
I'd think we'd want this to return `CustomerConfigurationData`, after updating it.